src/ui/Product.fs

```diff
@@ -0,0 +1,86 @@
+module Store.UI.Product
```

Comment:
Question: this is /product, will the URL after /product be generated based on the product name? (e.g. /product/float)

Reply:
Kind of, but it actually works the other way around. `/product/{anything here}` is the route, but it will refuse to load and show a page not found error if the `{anything here}` isn't a valid product name.